eng/common/pipelines/templates/steps/get-queuing-user.yml

```diff
@@ -0,0 +1,34 @@
+parameters: 
+  TargetVariable: ''
+
+steps:
+  # pull down identity-resolver here.
+  - pwsh: |
+      git clone https://github.com/Azure/azure-sdk-tools.git $(Build.SourcesDirectory)/tools_repo
+      cd $(Build.SourcesDirectory)/tools_repo
+      git checkout feature/identity-resolver
+    displayName: Get Appropriate Build Tools
```

Comment:
I want to replace this with a call to either.

A) a release
B) a public (but still not nuget.org) nuget feed install

Reply:
I found [this build](https://github.com/Azure/azure-sdk-tools/blob/master/eng/pipelines/tools-repo-versioning.yml)

So I can conceivably target a tag here. This is a chicken and egg problem though. Until this change merges, I won't have a tag that includes the code changes in this PR.